js/src/interface.js

```diff
@@ -274,6 +266,29 @@ let app = new Vue({
     }
   },
   methods: {
+    initDataSourceProvider(currentDataSourceId) {
+      let dataSourceData = {
+        dataSourceTitle: settings.dataSourceLabel ? settings.dataSourceLabel : 'Select data source',
```

Comment:
Let's use the same variable name unless you have any objections.

```suggestion
        dataSourceTitle: settings.dataSourceTitle ? settings.dataSourceTitle : 'Select data source',
```